plugins/modules/vmadm.py

```diff
@@ -432,7 +432,7 @@ def get_vm_uuid(module, alias):
             msg='Could not retrieve UUID of {0}'.format(alias), exception=stderr)
 
     # If no VM was found matching the given alias, we get back an empty array.
-    # That is not an error condition as we might be explicitly checking it's
+    # That is not an error condition as we might be explicitly checking its
```

Comment:
I believe it is slightly more correct as:
```suggestion
    # That is not an error condition as we might be explicitly checking for its
```
but I might be wrong.